www/front_src/src/Resources/Listing/index.test.tsx

```diff
@@ -371,7 +373,9 @@ describe(Listing, () => {
 
     const chipLabel = `${acknowledgedEntity?.name} ${labelAcknowledged}`;
 
-    const chip = await findByLabelText(chipLabel);
+    const chip = await findByLabelText(chipLabel, undefined, {
+      timeout: 10000,
```

Comment:
Jest tears down the test after 5 seconds, maybe the timeout for 10 seconds is not very relevant

Reply:
We changed the default timeout to 10sec to avoid ci failures

Reply:
why only the CI?  and why just x2 from the default?

Reply:
It’s not just a change within the CI, it’s global, but the failures manifested themselves mostly on the CI